### PR TITLE
Add version.h and .main.cpp.swp to .gitignore

### PR DESCRIPTION
**Pull request by myclevorname**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,6 +94,7 @@ clean:
 		rm -f $(OBJ) $(EXE) description-pak spasm-ng*.deb spasm-ng*.tar.gz
 		rm -f opt static prep-special-build
 		rm -f *.gcno *.gcda *.gcov
+		rm -r version.h
```
Comment (tari) [inline Makefile]: Probably want -f here rather than -r, since this isn't expected to be a directory.

Comment (myclevorname) [reply Makefile]: I fixed it in my latest commit.

**Comment by myclevorname** [comment]
PS: Also lays out the groundwork for the -q flag. It is only in main.cpp, and I haven't tested it.

**Comment by commandz0** [comment]
You should create a separate pull request if you want to implement multiple different features, that way they can be accepted individually if necessary.

File: main.cpp [hunk 2]
```diff
@@ -134,7 +134,8 @@ int run_assembly()
 	//...and if there's output, run the second pass and write it to the output file
 	if (mode & MODE_SYMTABLE || mode & MODE_NORMAL || mode & MODE_LIST)
 	{
-		printf ("Pass two... \n");
+		if (quiet_output)
```
Comment (commandz0) [inline main.cpp:137]: Why is this `quiet_output` instead of `!quiet_output`?

File: main.cpp [hunk 3]
```diff
@@ -270,6 +272,7 @@ int main (int argc, char **argv)
 	//init stuff
 	mode = MODE_NORMAL;
 	in_macro = 0;
+	quiet_output = true;
```
Comment (commandz0) [inline main.cpp:275]: Wouldn't this cause `quiet_output` to be true even if the `-q` flag is unset?

Comment (myclevorname) [reply main.cpp:275]: Oops...

**Comment by myclevorname** [comment]
I will delete this fork and start over.

**Pull request closed without merging**